### ISSUE TITLE
Enhancement: Non-zero-centeret prior in correlation

### ISSUE THREAD

**Issue by lindeloev** [jasp-stats/jasp-desktop#2817]
It would be nice to add informed priors to Bayesian correlation. For example, setting both alpha and beta in a stretched beta distribution. Currently, only zero-centered priors are possible.

Today, I taught Bayesian stats using JASP. It's generally a really nice experience (thanks!), but the question rightly came up how to set priors when positive correlations are expected a priori.

I would also make one more case for the ability to set priors before showing posteriors (#1995). Would be excellent for teaching!

**Comment by AlexanderLyNL** [comment]
I agree with showing priors before posteriors, we have been doing a lot of background work for that. Regarding informed priors to Bayesian correlations, this can be numerically, but not with a (stretched) beta prior. Shifting the centre with a beta prior results in a simultaneous change of the variance. I can decouple this with another prior, but I need to study its properties a bit more. Hence, this takes a bit of research, but not sure whether it's worth a paper. 

The good news is, is that JASP can do one-sided tests, just press BF_{+0} or BF_{-0}. I'll get back to both issues, but am focussing on some other issues at the moment.

**Comment by AlexanderLyNL** [comment]
Thanks for the encouraging words by the way. By other issues I meant the background work to eventually display the priors amongst other things. Let us know how your experience was teaching using JASP and whether there are other features you think are helpful for your course.

**Comment by lindeloev** [comment]
Great; I see the issue with the correlation prior and I'm looking forward to the a priori prior plot.

There are so many positive things to say about JASP as compared to SPSS that I don't know where to start, but the uncluttered interface, output, and editing of analyses really hits home. Ease of installation and the demo datasets too.

It's dangerous asking for further ideas and experiences, so please bear with me! Other ideas which would give me a leaner teaching experience (most important first):

 * I just updated #1260 with my primary wish for JASP for teaching right now. My course is centered around linear models and the realization that all named models (t-test, ANOVA, etc.) are just special cases of linear models, using dummy coding etc. This puts the main focus on estimation and simultaneously prepares the students for putting priors on parameters.

* Graph builder and customizable plots, as has been suggested a million times, but I know how big a development effort that would be! This is one aspect where I actually think SPSS does pretty well.

* GLMMs (#1103, Jamovi's GALMj is pretty good but SPSS is also decent here). If I could only share the joy of using `lme4::lmer` and `brms::brm` with my students in a more accessible manner :-)

* Hedge's g in t-test and post-hoc tests (#2270). It is generally preferrable over Cohen's d.

* I also still get questions why there are two menus for Bayesian correlation (#1875) - students get the impression that they are different *kinds* of correlation.

* Random effects in meta-analysis to allow for analysis of pre/post designs with treatment/control. I'm still using OpenMetaAnalyst for this particular (but very common) case. Jamovi's MAJOR does not have this either but it does have meta-analysis of correlation, which is great.

**Comment by AlexanderLyNL** [comment]
Thanks for your input, I'll discuss this with the team.
